components/PicklistBase/PicklistBase.content.comp.cy.js

```diff
@@ -11,283 +11,156 @@ describe('Test the default PicklistBase content', () => {
     cy.saveSessionStorage();
   });
 
-  it('Check all of the default data-cy elements', () => {
-    const readySpy = cy.spy().as('readySpy');
-
+  it('Check initial header button state with checkbox mode', () => {
     cy.mount(PicklistBase, {
       props: {
-        onReady: readySpy,
-        rows: [],
+        rows: [
+          { name: 'Item A', quantity: 5, location: 'GHANA' },
+          { name: 'Item B', quantity: 3, location: 'GHANA' },
+        ],
         columns: ['name', 'quantity', 'location'],
         labels: { name: 'Name', quantity: 'Quantity', location: 'Location' },
+        picked: new Map(),
+        showAllButton: true
       },
     });
 
-    cy.get('@readySpy')
-      .should('have.been.calledOnce')
-      .then(() => {
-        cy.get('[data-cy="picklist-table"]').should('exist');
-        cy.get('[data-cy="picklist-header-name"]').should('exist');
-        cy.get('[data-cy="picklist-header-quantity"]').should('exist');
-        cy.get('[data-cy="picklist-header-location"]').should('exist');
-        cy.get('[data-cy="picklist-invalid-feedback"]').should('not.exist');
-        cy.get('[data-cy="picklist-sort-button-name"]')
-          .should('exist')
-          .and('contain.text', 'Name');
-        cy.get('[data-cy="picklist-sort-button-quantity"]')
-          .should('exist')
-          .and('contain.text', 'Quantity');
-        cy.get('[data-cy="picklist-sort-button-location"]')
-          .should('exist')
-          .and('contain.text', 'Location');
-      });
+    cy.get('[data-cy="picklist-all-button"]')
```

Comment:
This looks like the right test. But it won't pass because the icon doesn't exist in the button when in _checkbox_ mode (i.e. when not using `units`).  See the comment in `PicklistBase.vue`.